examples/knative/python/kn-py-slack/README.md

```diff
@@ -0,0 +1,117 @@
+# kn-ps-slack
```

Comment:
```suggestion
# kn-py-slack
```

Reply:
made the change and signed commit 9d9afc8.